src/App.tsx

```diff
@@ -2,108 +2,34 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import { CommentTypes, UserTypes, PostTypes } from './types';
 
-export const App: React.FC = () => (
-  <section className="App">
-    <h1 className="App__title">Static list of posts</h1>
-
-    <div className="PostList">
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">qui est esse</h3>
-
-          <p>
-            {' Posted by  '}
-
-            <a className="UserInfo" href="mailto:Sincere@april.biz">
-              Leanne Graham
-            </a>
-          </p>
-        </div>
-
-        <p className="PostInfo__body">
-          est rerum tempore vitae sequi sint nihil reprehenderit dolor beatae
-          ea dolores neque fugiat blanditiis voluptate porro vel nihil
-          molestiae ut reiciendis qui aperiam non debitis possimus qui neque
-          nisi nulla
-        </p>
-
-        <hr />
-
-        <b data-cy="NoCommentsMessage">No comments yet</b>
-      </div>
-
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">
-            doloremque illum aliquid sunt
-          </h3>
-
-          <p>
-            {' Posted by  '}
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { PostList } from './components/PostList';
 
-            <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-              Patricia Lebsack
-            </a>
-          </p>
-        </div>
-
-        <p className="PostInfo__body">
-          deserunt eos nobis asperiores et hic est debitis repellat molestiae
-          optio nihil ratione ut eos beatae quibusdam distinctio maiores earum
-          voluptates et aut adipisci ea maiores voluptas maxime
-        </p>
-
-        <div className="CommentList">
-          <div className="CommentInfo">
-            <div className="CommentInfo__title">
-              <strong className="CommentInfo__name">pariatur omnis in</strong>
-
-              {' by '}
-
-              <a
-                className="CommentInfo__email"
-                href="mailto:Telly_Lynch@karl.co.uk"
-              >
-                Telly_Lynch@karl.co.uk
-              </a>
-            </div>
-
-            <div className="CommentInfo__body">
-              dolorum voluptas laboriosam quisquam ab totam beatae et aut
-              aliquid optio assumenda voluptas velit itaque quidem voluptatem
-              tempore cupiditate in itaque sit molestiae minus dolores magni
-            </div>
-          </div>
+const getComments = (id: number) => (commentsFromServer
+  .filter((comment: CommentTypes) => comment.postId === id)
+);
```

Comment:
```suggestion
const getComments = (id: number) => (
  commentsFromServer.filter(
    (comment: Comment) => comment.postId === id
  )
);
```

Reply:
@jstmpelowycz @amitpatiljc After implement this I have error missing trailing comma (which is weird) because it's not object

![Untitled](https://user-images.githubusercontent.com/86070599/195370079-f91748f2-8f50-46e3-9cbd-73b5e0dd8417.png)

Reply:
![Screenshot 2022-10-12 at 17 29 47](https://user-images.githubusercontent.com/75496546/195370608-fe1cdf78-8d98-4686-805b-498fbc4b565a.png)
this is the expected behavior, we can pass other arguments to`Array.prototype.filter`
also, FYI, trailing commas should be put not only in objects

Reply:
So I changed like this
const getComments = (id: number) => (
  commentsFromServer.filter((comment: Comment) => (
    comment.postId === id
  ))
);